src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -553,16 +553,7 @@ private static string GetNextArgument(string arguments, ref int i)
                 // it contains spaces.
                 if (c == '"')
                 {
-                    if (inQuotes && i < arguments.Length - 1 && arguments[i + 1] == '"')
-                    {
-                        currentArgument.Append('"');
-                        i++;
-                    }
-                    else
-                    {
-                        inQuotes = !inQuotes;
-                    }
-
+                    inQuotes = !inQuotes;
```

Comment:
requires adding more tests, and and updating the existing test cases, if none of the test cases broke with this change, then there was not sufficient test cases to begin with.